vnext/Microsoft.ReactNative/Views/FrameworkElementViewManager.cpp

```diff
@@ -526,29 +526,78 @@ bool FrameworkElementViewManager::UpdateProperty(
       DynamicAutomationProperties::SetAccessibilityStateCollapsed(
           element, states[static_cast<int32_t>(winrt::Microsoft::ReactNative::AccessibilityStates::Collapsed)]);
     } else if (propertyName == "accessibilityValue") {
+      auto textValue = static_cast<winrt::hstring>(L"");
+      double values[static_cast<int32_t>(winrt::Microsoft::ReactNative::AccessibilityValue::CountValues) - 1] = {};
```

Comment:
Conceptually, this isn't CountValues-1, but "# of double-valued properties". If we added another enum for storing some string, CountValues would be 5, but we would still have 3 number values